R/data_accessors.R

```diff
@@ -71,24 +68,14 @@ get_mpx_cases <- function(path, connection = NULL, include_endemic = TRUE) {
   # Pull data using whatever method required
   raw_data <- fetch_cdc_mpx_data(path, connection)
```

Comment:
@kimkimroll @jamesfuller-cdc with the new changes removing the pivot and processing, `get_mpx_cases()` is basically just `fetch_cdc_mpx_data()`, with the addition of the endemic_country filter. 

If we ever drop that filter, we should just delete get_mpx_cases/deaths and just use fetch_cdc_mpx_data() for simplicity.

Reply:
@beansrowning U right! I'll try to get to cleaning this, but might save for Spring Cleaning time

Reply:
Yeah let's save for now because we're still using the endemic/non-endemic filter for stuff on the app.